src/redux/store.js

```diff
@@ -1,12 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit';
 import logger from 'redux-logger';
-import missionsReducer from './redux/missions';
+import thunk from 'redux-thunk';
+import missionsReducer from './missions';
+import rocketsReducer from './rockets/rockets';
```

Comment:
- [ ] The path for the `missions.js` is creating a conflict.
Would be good to take a look at the `develop` branch and try to merge it into your feature branch, so you will be able to update the right path for this file.